net/Socket.cpp

```diff
@@ -824,8 +825,33 @@ bool ServerSocket::bind(Type type, int port)
 
     int rc;
 
-    assert (_type != Socket::Type::Unix);
-    if (_type == Socket::Type::IPv4)
+    if (_type == Socket::Type::Unix)
+    {
+        struct sockaddr_un addrunix;
+
+        _socketPath = std::string(Socket::SocketPath);
+        int last_errno = 0;
+        std::memset(&addrunix, 0, sizeof(addrunix));
+        addrunix.sun_family = AF_UNIX;
+        std::memcpy(addrunix.sun_path, _socketPath.c_str(), _socketPath.length());
```

Comment:
The paranoid in me wants to check that length against sizeof sun_path and fail elegantly if too long =)